lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -248,7 +249,10 @@ export class PublicClientApplication extends ClientApplication implements IPubli
                             requestWithCLP.cacheLookupPolicy === CacheLookupPolicy.AccessTokenAndRefreshToken ||
                             requestWithCLP.cacheLookupPolicy === CacheLookupPolicy.RefreshToken)
                         && (requestWithCLP.cacheLookupPolicy !== CacheLookupPolicy.Skip)
+                        && !attemptSsoRTNotFound
                     ) {
+                        // eslint-disable-next-line no-console
+                        console.log("acquireTokenSilentAsync: retrieving refresh token failed.");
```

Comment:
```suggestion
```

Reply:
Yeah, removing this, was trying to test something and forgot to remove it. Thanks.